src/activities/assignments/AssignmentEntity.js

```diff
@@ -188,7 +188,13 @@ export class AssignmentEntity extends Entity {
 			return null;
 		}
 
-		return this._entity.properties.defaultScoringRubricId;
+		const defaultScoringRubricId = this._entity.properties.defaultScoringRubricId;
+
+		if (defaultScoringRubricId === undefined || defaultScoringRubricId === null) {
```

Comment:
Might be possible that ID is 0? 🤷

Reply:
If it's just rubrics, doesn't look like 0 is actually valid. Looks like it starts at 1: https://search.d2l.dev/xref/lms/lp/_dbschema/D2L.LP.Rubrics/Tables/RUBRICS.sql?r=55de0cab#5